shared/components/Location.tsx

```diff
@@ -49,38 +50,44 @@ function Location(props: {location : string}){
         })
     }
 
+    const hasColDiv = (props.location == "brandywine")
     //Display HTML
     return (
         <View style={styles.location}>
-            <LocationHeader locationInfo={locationInfo}/>
-            <View style={styles.stationList}>
-                <Text>{loadingMessage}</Text>
-                {stationInfo}
+            <View style={{width: hasColDiv ? "calc(100% - 4px)" : "100%"}}>
+                <LocationHeader locationInfo={locationInfo}/>
+                <View style={styles.stationList}>
+                    <Text>{loadingMessage}</Text>
+                    {stationInfo}
+                </View>
             </View>
+            <View style={[styles.columnDivider, {width: hasColDiv ? "4px" : "0px"}]}></View>
         </View>
     )
 }
 
 function LocationHeader(props: {locationInfo: LocationInfo}){
     const info = props.locationInfo
     let currMeal = info.currentMeal
+    let location = info.restaurant
+
+    if(location)
+        location = location.charAt(0).toUpperCase() + location.slice(1)
+
     if (currMeal)
         {currMeal = currMeal.charAt(0).toUpperCase() + currMeal.slice(1)}
 
     let locationUrl = "https://www.google.com/maps/search/uci+"
 
     return(
-        <View style={headerStyles.locationHeader}>
+        <ImageBackground source={"components/imageAssets/" + location + ".imageset/" + location + ".jpg"}
```

Comment:
Looks like these aren't rendering on the preview deployment with 404's. You can reproduce the issue locally with `npm run build` and serving the `dist` folder with something like `python -m http.server` from inside `dist`. I think the problem has something to do with Vite not automatically including the asset because these source paths are dynamically generated, so you might have to figure out a way to force it to include the assets in the build, or figure out a location to move them to specifically for static assets that Vite will always include in the build.

Reply:
I moved the components symlink to the public folder in zotmeal-vite which now allows the build version to load all image assets